### ISSUE TITLE
Import wallet by private key sometimes take long time

### ISSUE THREAD

**Issue by seabornlee** [AlphaWallet/alpha-wallet-android#2966]
![Screenshot 2022-11-22 at 22 18 48](https://user-images.githubusercontent.com/735708/203337342-d38c1346-44cb-40aa-9aee-34c1a3f3c6a9.png)

**Comment by seabornlee** [comment]
![Screenshot 2022-11-22 at 22 18 48](https://user-images.githubusercontent.com/735708/203337342-d38c1346-44cb-40aa-9aee-34c1a3f3c6a9.png)

**Comment by JamesSmartCell** [comment]
@seabornlee I see this all the time if I have the USB cable attached and I have been debugging. Can you see if it still happens if you unplug the USB cable (or switch off wifi debugging)?

**Comment by seabornlee** [comment]
Yes, it's happening in emulator, tests sometimes failed because of this.
